src/components/verification-grid/css/style.css

```diff
@@ -15,6 +15,11 @@
   pointer-events: none;
 }
 
+#skip-button {
+  height: min-content;
```

Comment:
yeah not sure on this

Reply:
Mock-up without the min-content height

![image](https://github.com/user-attachments/assets/1a45ade1-a391-44b0-b9b9-5a6658dc6d95)

This is how it originally looked, but I (personally) decided that it looks bad

(Ignore the un-evenly sized buttons I'm trying out something from a previous comment, but haven't finished yet)